dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -390,8 +391,26 @@ public Service initializeServiceFromGitHub(String repositoryId, String subclass)
      * @param workflowName Name of the workflow
      * @return Workflow
      */
-    public BioWorkflow initializeWorkflowFromGitHub(String repositoryId, String subclass, String workflowName) {
```

Comment:
I would have kept this returning a BioWorkflow, see comment below on how you could have done it.

The caller just casts it to a `Workflow` anyway, so no need to fix, but FYI.